### PR TITLE
Audit Chore: Linter emits warning

### LINKED ISSUE #187: Audit G4: Linter emits 25 warnings
What Audit Says:
image

### PR DESCRIPTION
**Pull request by Zitzak**
## What did I do?

- solved all the linter warnings where possible
- silenced the ones I couldn't solve

## Implements issues:
- #187 

closes #187

### CODE REVIEW

**Review by codebuster22 — DISMISSED** [review]
lgtm

File: contracts/lbp/LBPManager.sol [hunk 1]
```diff
@@ -21,6 +21,7 @@ import "../utils/interface/ILBP.sol";
  * @title LBPManager contract.
```
[the remark this answers is not in the document]
Comment (Zitzak) [reply contracts/lbp/LBPManager.sol:21]: Solved

**Review by fabianschu — DISMISSED** [review]
🚀

**Review by BritikovKI — APPROVED** [review]
To the moon :rocket:

**Review by fabianschu — APPROVED** [review]

**Pull request merged**